includes/class-shortcodes.php

```diff
@@ -444,17 +444,17 @@ public function sc_listings_featured( $atts ) {
 	 * @since x.x
 	 */
 	private function process_category_atts( $atts, &$query_args ) {
-		if ( ! isset( $atts['category'] ) && ! isset( $atts['categories'] ) ) {
+		if ( ! $atts['category'] && ! $atts['categories'] ) {
```

Comment:
It looks like 'categories' won't always be set so we'll still need to check for it to avoid php warnings.

Reply:
I have added the array checks and placed some conditional checks to prevent returning or setting values if nothing is returned